packages/neos-ui-contentrepository/src/registry/NodeTypesRegistry.js

```diff
@@ -163,4 +163,17 @@ export default class NodeTypesRegistry extends SynchronousRegistry {
 
         return viewConfiguration;
     }
+
+    getInlineEditorForProperty(nodeTypeName, propertyName) {
+        const nodeType = this.get(nodeTypeName);
+
+        return $get(['properties', propertyName, 'ui', 'inline', 'editor'], nodeType) || 'ckeditor';
```

Comment:
I think it's best to leave a `TODO` there and not document it at all for now, since I believe this deserves further discussion before becoming official.

Documenting it as publicly as in the README might suggest it likely to become public API. But since it cannot be considered stable until there is a certain level of confidence within the team, I'd prefer to avoid confusion of that kind.

However, I think it's justifiable to create an example within https://github.com/neos/neos-ui-extensibility-examples to showcase the general concept for people to give it a try. wdyt?